### ISSUE TITLE
[Feature Request] Payment gateway outside of the app store

### ISSUE THREAD

**Issue by mystic-githuber** [olvid-io/olvid-android#10]
Create a payment gateway outside of the app store  on your website for example like Zoom or Threema as some of us use ROMs and can't access the store or would like to have a desktop only experience. 

Perhaps a phone credit like system where we can top-up our account also is a good idea.

**Comment by Naia-love** [comment]
Yes please, its the only case where I may actually pay the subs (despite an imo too high price, plz we need 1/2€ subs)
Mullvad system work very well and its nice

**Comment by PLAYERUNKNOWNHatsune01** [comment]
Bump! Me too. I don't mind the price (even though it's 4$ a month) I would like to pay outside the playstore! Because I'm stick of Google BS and planning to change ROM too. And talking about outside the store can we download Olvid app outside of the playstore too.

**Comment by stemy2** [comment]
Olvid is now available on F-droid in a version where the subscriptions button is still present but isn't doing anything. This make an external payment even more crucial.
